### ISSUE TITLE
Somethings wrong with the maths

### ISSUE THREAD

**Issue by diversemix** [diversemix/calc-hurricane-loss#1]
I've implemented a C++ version as well - but there seems to be wildly different results in the answer 

Help: I need a mathematician!? :nerd_face: 

/cc @mpinkerton-oasis @sambles ?

```
$ time gethurricaneloss -n 1000000 10 2.2 0.2 22 1.1 0.1 ; time ./loss-calculator/build/loss-calculator 
Mean loss = 158.570 per year calculated over 1000000 years.

real  0m22.210s
user 0m22.322s
sys  0m0.220s
Mean loss=184.115 per year calculated over 1000000 years.

real  0m4.470s
user 0m4.463s
sys  0m0.004s
```

**Comment by diversemix** [comment]
I am thinking this is down to rounding errors in python. :disappointed: 

Links to the relevant cpp docs:
https://en.cppreference.com/w/cpp/numeric/random/poisson_distribution
https://en.cppreference.com/w/cpp/numeric/random/lognormal_distribution

**Comment by diversemix** [comment]
Fixed - this was an issue in the cpp code using the same region twice rather than the two different regions :man_facepalming: - Need tests!
